pkg/flow/reconciler/xslttransform/adapter.go

```diff
@@ -84,5 +86,11 @@ func makeAppEnv(o *v1alpha1.XSLTTransform) ([]corev1.EnvVar, error) {
 		})
 	}
 
+	if sink != nil {
+		env = append(env, corev1.EnvVar{
+			Name:  envSink,
+			Value: sink.Path,
```

Comment:
Did you mean `sink.String()`? Otherwise this value is `/`, which is not going to work.

Reply:
good catch!